hedera-node/configuration/mainnet/api-permission.properties

```diff
@@ -0,0 +1,56 @@
+# Crypto
+createAccount=0-*
+cryptoTransfer=0-*
+updateAccount=0-*
+cryptoGetBalance=0-*
+getAccountInfo=0-*
+cryptoDelete=0-*
+getAccountRecords=0-*
+getTxRecordByTxID=0-*
+getTransactionReceipts=0-*
+# File
+createFile=0-*
+updateFile=0-*
+deleteFile=0-*
+appendContent=0-*
+getFileContent=0-*
+getFileInfo=0-*
+# Contract
+createContract=0-*
+updateContract=0-*
+contractCallMethod=0-*
+getContractInfo=0-*
+contractCallLocalMethod=0-*
+contractGetBytecode=0-*
+getTxRecordByContractID=0-*
+deleteContract=0-*
+# Consensus
+createTopic=0-*
+updateTopic=0-*
+deleteTopic=0-*
+submitMessage=0-*
+getTopicInfo=0-*
+# Scheduling
+scheduleCreate=0-*
+scheduleSign=0-*
+scheduleDelete=0-*
+scheduleGetInfo=0-*
+# Token
+tokenCreate=0-*
+tokenFreezeAccount=0-*
+tokenUnfreezeAccount=0-*
+tokenGrantKycToAccount=0-*
+tokenRevokeKycFromAccount=0-*
+tokenDelete=0-*
+tokenMint=0-*
+tokenBurn=0-*
+tokenAccountWipe=0-*
+tokenUpdate=0-*
+tokenGetInfo=0-*
+tokenAssociateToAccount=0-*
+tokenDissociateFromAccount=0-*
+# Network 
+getVersionInfo=0-*
+systemDelete=2-59
+systemUndelete=2-60
+freeze=2-58
```

Comment:
Is it not required to have throttles set for `hedera-node/configuration/mainnet/bootstrap.properties`  or will it be updated by devops?